### PR TITLE
Fix cert signing in local bench

### PR DESCRIPTION
**Pull request by sadhansood**

### CODE REVIEW

**Review by longbowlu — APPROVED** [review]
> // TODO: Why iterating from 0 to quorum_threshold??

so why?

**Review by huitseeker — CHANGES_REQUESTED** [review]
IIUC, iterating through 67% of the stake should be enough to get a valid vote, and the `quorum_threshold` should abstract over that 67% threshold. You should not have to iterate through the full committee.

We should find what the issue is and open a bug / fix it.

**Comment by lxfind** [comment]
Let's fix this TODO. Basically we should iterate until the accumulated stake is at quorum threshold

**Review by huitseeker — APPROVED** [review]
I can go for that!

File: crates/sui-benchmark/src/benchmark/transaction_creator.rs [hunk 1]
```diff
@@ -62,15 +62,22 @@ fn make_cert(network_config: &NetworkConfig, tx: &Transaction) -> CertifiedTrans
...
             .get(i as usize)
             .unwrap()
             .key_pair();
+        
         let pubx: AuthorityPublicKeyBytes = secx.public().into();
         let sig = AuthoritySignature::new(&tx.data, secx);
+        let authority_weight = committee.weight(&pubx);
         sigs.push((pubx, sig));
+        total_stake += authority_weight;
+        if total_stake > committee.quorum_threshold() {
```
Comment (lxfind) [inline crates/sui-benchmark/src/benchmark/transaction_creator.rs:78]: >=

File: crates/sui-benchmark/src/benchmark/transaction_creator.rs [hunk 2]
```diff
@@ -62,15 +62,22 @@ fn make_cert(network_config: &NetworkConfig, tx: &Transaction) -> CertifiedTrans
     let committee = network_config.committee();
     let mut sigs: Vec<(AuthorityName, AuthoritySignature)> = Vec::new();
     // TODO: Why iterating from 0 to quorum_threshold??
```
Comment (lxfind) [inline crates/sui-benchmark/src/benchmark/transaction_creator.rs:64]: Can remove TODO now

File: crates/sui-benchmark/src/benchmark/transaction_creator.rs (hunk 1 above)
Comment (sadhansood) [reply crates/sui-benchmark/src/benchmark/transaction_creator.rs:78]: ooo.. good catch

File: crates/sui-benchmark/src/benchmark/transaction_creator.rs (hunk 2 above)
Comment (sadhansood) [reply crates/sui-benchmark/src/benchmark/transaction_creator.rs:64]: for sure!

**Review by lxfind — APPROVED** [review]

**Comment by sadhansood** [comment]
> so why?

I'm not sure myself, this code predates me

**Pull request merged**